src/controllers/update/check.your.answers.controller.ts

```diff
@@ -0,0 +1,68 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import { getApplicationData } from "../../utils/application.data";
+import { ApplicationData } from "../../model";
+import { Session } from "@companieshouse/node-session-handler";
+import { isActiveFeature } from "../../utils/feature.flag";
+import { OverseasEntityKey, Transactionkey } from "../../model/data.types.model";
+
+import { createOverseasEntity, updateOverseasEntity } from "../../service/overseas.entities.service";
+import { closeTransaction, postTransaction } from "../../service/transaction.service";
+import { startPaymentsSession } from "../../service/payment.service";
+
+import {
+  CHS_URL,
+  FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022,
+  OVERSEAS_ENTITY_QUERY_PAGE,
+  UPDATE_AN_OVERSEAS_ENTITY_URL,
+  UPDATE_CHECK_YOUR_ANSWERS_PAGE
+} from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${UPDATE_CHECK_YOUR_ANSWERS_PAGE}`);
+
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    return res.render(UPDATE_CHECK_YOUR_ANSWERS_PAGE, {
+      backLinkUrl: OVERSEAS_ENTITY_QUERY_PAGE,
+      templateName: UPDATE_CHECK_YOUR_ANSWERS_PAGE,
+      appData
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${UPDATE_CHECK_YOUR_ANSWERS_PAGE}`);
+
+    const session = req.session as Session;
+    const appData: ApplicationData = getApplicationData(session);
+
+    let transactionID, overseasEntityID;
+    if (isActiveFeature(FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) {
```

Comment:
We should add 
 if (!isActiveFeature(config.FEATURE_FLAG_ENABLE_ROE_UPDATE) && isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) {

Reply:
I'm just thinking should we even have this `if` block at all? Just keep the `else` part?
 Since there is the existing controller for the register journey, and this new controller will only be used for update journey this code behind `!isActiveFeature(config.FEATURE_FLAG_ENABLE_ROE_UPDATE) && isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022))` wouldn't get called

Reply:
I've simplified the code block for now if there is upcoming save & resume epic.